File: src/BTech_Back/BTech.Domain/Model/Order.cs
public class Order
{
    public Guid Id { get; set; }
    public Guid UserId { get; set; }
    public decimal TotalValue { get; set; }
    public List<OrderItem> OrderItems { get; set; }

    public Order(Guid userId, decimal totalValue, List<OrderItem> orderItems)
    {
        Id = Guid.NewGuid();
        UserId = userId;
        TotalValue = totalValue;
        OrderItems = orderItems ?? new List<OrderItem>();
    }

    public Order()
    {
        OrderItems = new List<OrderItem>();
    }
}

File: src/BTech_Front/src/app/models/order.ts
export interface Order {
  id: string;
  name: string;
  productName: string;
  productImages?: string[]; // Adicionando a propriedade productImages
  date: string;
  status: string;
  userId: string;
  deliveryDate: Date;
  totalValue: number;
  orderItems: any[];
  merchantOrderId: string;
}
